pdh_json_updater/json_card.py

```diff
@@ -39,6 +39,9 @@ def is_legal(
         # check illegal card types
         if front_card_face_typeline in cls.ILLEGAL_CARD_TYPES:
             return Legality.NOT_LEGAL.value
+        # check illegal game types
```

Comment:
for arena sets (SIR), the "game" field is "game: 0: arena"
[link to json](https://api.scryfall.com/cards/1b606200-a4c0-4e89-b558-4cec72d4409f?format=json&pretty=true)